color.go

```diff
@@ -107,6 +112,27 @@ func (c RGBColor) RGBA() (r, g, b, a uint32) {
 //	colorB := lipgloss.ANSIColor(134)
 type ANSIColor = ansi.ExtendedColor
 
+// LightDark is a helper type that can be used to specify colors that should be
+// used based on the terminal's background color.
+//
+// Example usage:
+//
+//	useDark := lipgloss.LightDark(true)
+//	light := "#0000ff"
+//	dark := "#ff0000"
+//	colorToUse := useDark.Color(light, dark)
+//	fmt.Println(colorToUse)
+type LightDark bool
```

Comment:
What do you think about naming this `Adapt` instead? Might feel more familiar.

Then it could be 
```
adaptive := lipgloss.Adapt(true)
colors := adaptive.Color("#0000ff", "#ff0000")
```

I think that would make the purpose more clear

Reply:
Totally: this is really nice.

Reply:
I would also update the comment to show use with Bubble Tea, as well as standalone Lip Gloss (when we get there).

Reply:
I'm thinking of dropping this type which is basically an ifelse; the adaptive sub-package should be enough imo